src/common/routes.js

```diff
@@ -16,5 +17,6 @@ export default (
     <Route path="/:owner/:name/builds" component={Builds}/>
     <Route path="/:owner/:name/builds/:buildId" component={BuildDetails}/>
     <Route path="/login/failed" component={LoginFailed}/>
+    <Route path="/dashboard/select-repositories" component={SelectRepositories}/>
```

Comment:
Can you move this up a few lines so that it's next to `/dashboard`?

I think I'd prefer a less wordy name; `/dashboard/add` sounds OK.